nxdrive/client/local/linux.py

```diff
@@ -52,10 +65,24 @@ def remove_remote_id_impl(path: Path, name: str = "ndrive") -> None:
                 raise exc
 
     def set_folder_icon(self, ref: Path, icon: Path) -> None:
-        """Create a special file to customize the folder icon."""
+        """Use commandline to customize the folder icon."""
         log.debug(f"Setting the folder icon of {ref!r} using {icon!r}")
-        # To be implementation with https://jira.nuxeo.com/browse/NXDRIVE-1831
-        return
+        folder_path = self.abspath(ref).as_posix()
```

Comment:
On GNU/Linux, `.as_posix()` is not needed. You could then move that variable directly into the `subprocess` call.